bootstrap.py

```diff
@@ -465,8 +475,9 @@ class MEOptions:
     # doublecheck and copy it manually if not
     if not os.path.exists('/etc/pki/product/'):
         os.mkdir("/etc/pki/product/")
-    if not os.path.exists('/etc/pki/product/69.pem'):
-        for line in open("/usr/share/rhsm/product/RHEL-5/channel-cert-mapping.txt"):
+    mapping_file = "/usr/share/rhsm/product/RHEL-5/channel-cert-mapping.txt"
+    if not os.path.exists('/etc/pki/product/69.pem') and os.path.exists(mapping_file):
+        for line in open(mapping_file):
             if ("rhel-x86_64-server-5:" in line and ARCHITECTURE == "x86_64") or ("rhel-i386-server-5:" in line and ARCHITECTURE == "x86"):
```

Comment:
as you fixed the content of `ARCHITECTURE`, I think this should be something like:

    if line.startswith('rhel-%s-server-5' % ARCHITECTURE):
    …